src/mono/wasm/debugger/BrowserDebugProxy/MonoSDBHelper.cs

```diff
@@ -0,0 +1,1872 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System.Net.Http;
+using System.Text.RegularExpressions;
+
+namespace Microsoft.WebAssembly.Diagnostics
+{
+    internal enum TokenType
+    {
+        mdtModule               = 0x00000000,       //
+        mdtTypeRef              = 0x01000000,       //
+        mdtTypeDef              = 0x02000000,       //
+        mdtFieldDef             = 0x04000000,       //
+        mdtMethodDef            = 0x06000000,       //
+        mdtParamDef             = 0x08000000,       //
+        mdtInterfaceImpl        = 0x09000000,       //
+        mdtMemberRef            = 0x0a000000,       //
+        mdtCustomAttribute      = 0x0c000000,       //
+        mdtPermission           = 0x0e000000,       //
+        mdtSignature            = 0x11000000,       //
+        mdtEvent                = 0x14000000,       //
+        mdtProperty             = 0x17000000,       //
+        mdtModuleRef            = 0x1a000000,       //
+        mdtTypeSpec             = 0x1b000000,       //
+        mdtAssembly             = 0x20000000,       //
+        mdtAssemblyRef          = 0x23000000,       //
+        mdtFile                 = 0x26000000,       //
+        mdtExportedType         = 0x27000000,       //
+        mdtManifestResource     = 0x28000000,       //
+        mdtGenericParam         = 0x2a000000,       //
+        mdtMethodSpec           = 0x2b000000,       //
+        mdtGenericParamConstraint = 0x2c000000,
+
+        mdtString               = 0x70000000,       //
+        mdtName                 = 0x71000000,       //
+        mdtBaseType             = 0x72000000,       // Leave this on the high end value. This does not correspond to metadata table
+    }
+
+    internal enum CommandSet {
```

Comment:
I think we have 3 different enum value casings in this now, is there a reason to avoid using CamelCase for all of them? Is the idea to make the comparison to the C code a little more direct?

Reply:
I just copied from debugger-libs. Changed to CamelCase.